packages/sources/por-address-list/src/index.ts

```diff
@@ -8,6 +8,13 @@ export const adapter = new PoRAdapter({
   name: 'POR_ADDRESS_LIST',
   config,
   endpoints: [address, solvBTC, bedrockBTC, multichainAddress],
+  rateLimiting: {
```

Comment:
Why is this needed?

Reply:
otherwise blasting rest api endpoints with as many req/s as possible

Reply:
updated to 1/s instead